app/controllers/application_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
-  before_action :authenticate_user!
+  #before_action :authenticate_user!
```

Comment:
like we talked before, we would want to leave this here and skip the before action in the `static_controller`. See comment there.